pkg/broker/broker.go

```diff
@@ -769,3 +769,17 @@ func (a AnsibleBroker) RemoveSpecs() error {
 	}
 	return nil
 }
+
+func ocLogin(log *logging.Logger, args ...string) error {
+	log.Debug("Logging into openshift...")
+
+	fullArgs := append([]string{"login"}, args...)
+
+	output, err := apb.RunCommand("oc", fullArgs...)
+
+	if err != nil {
+		log.Debug(string(output))
+		return err
+	}
+	return nil
+}
```

Comment:
Not sure yet, setting up an origin client hello world is on my TODO list, I've been unsuccessful at finding an example.

More cluster specifics we'll need to abstract away too. `client.Login` interface method?